API/block/README.md

```diff
@@ -0,0 +1,78 @@
+block API
+=========
+
+#### `get`
+
+> Get a raw IPFS block.
+
+##### `Go` **WIP**
+
+##### `JavaScript` - ipfs.block.get(multihash, [callback])
+
+`multihash` is a [multihash][] which can be passed as
+
+- Buffer, the raw Buffer of the multihash 
+- String, the base58 encoded version of the multihash
+
+`callback` must follow `function (err, block) {}` signature, where `err` is an error if the operation was not successful and `block` is a [Block][].
+
+
+```js
+ipfs.block.get(multihash, function (err, data) {
+  // data is the raw data contained in a block
+})
+```
+
+If no `callback` is passed, a promise is returned.
+
+
+
+
+#### `put`
+
+> Stores input as an IPFS block.
+
+##### `Go` **WIP**
+
+##### `JavaScript` - ipfs.block.put(data, [callback])
+
+Where `data` can be a
+
+- Buffer, requiring that the encoding is specified on the options. if no 
```

Comment:
options are mentioned here, but the function signature doesn't have a field for options.